crimsobot/cogs/utilities.py

```diff
@@ -14,6 +16,111 @@ class Utilities(commands.Cog):
     def __init__(self, bot: CrimsoBOT):
         self.bot = bot
 
+    @commands.command(brief='Create a poll!')
+    async def poll(self, ctx: commands.Context, *, input: str) -> None:
+        """Make a poll! Use the form of some question;option 1;option 2;etc.
+        For example:
+
+        >poll What should I eat?;tacos;ramen;the rich
+
+        Polls can have up to 20 choices, but keep in mind Discord's 2000-character message limit.
+        Polls technically never close, but you can >tally the results whenever you like!
+        """
+
+        poll_pool = [
+            '1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣', '🔟',
+            '🇦', '🇧', '🇨', '🇩', '🇪', '🇫', '🇬', '🇭', '🇮', '🇯'
+        ]
+
+        choices = input.split(';')
+        question = choices.pop(0)
+
+        poll_body = ['**{}**\n'.format(question.strip())]
+        for idx, choice in enumerate(choices):
+            poll_body.append('{} {}'.format(poll_pool[idx], choice.strip()))
+
+        embed = c.crimbed(
+            title='{} has created a poll!'.format(ctx.author),
+            descr='{}'.format('\n'.join(poll_body)),
+            thumb_name='think',
+            footer='\u200dPoll ID: {d.month}{d.day}{d.hour}{d.minute}{d.second}'.format(d=datetime.utcnow())
```

Comment:
only real issue i see here is conflicts between polls created in the same second

Reply:
I uhhhh will take my chances until I come up with something more clever. (I'm gonna eventually cave to using a db I just know it)